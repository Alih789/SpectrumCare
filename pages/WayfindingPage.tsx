import React from 'react';
import { Text, SafeAreaView, StyleSheet, View, FlatList, TouchableHighlight } from 'react-native';

import Ionicons from 'react-native-vector-icons/Ionicons';

import { useNavigation } from '@react-navigation/native';

type RouteProps = { title: string, routeID: string };

function Route({ title, routeID }: RouteProps) {

  const navigation = useNavigation();

  return (
    <View >
      <TouchableHighlight
        activeOpacity={0.6}
        underlayColor="#DDDDDD"
        onPress={() => {
          navigation.navigate('Wayfinding', {
            screen: 'Route',
            params: {
              routeID: routeID,
              routeTitle: title,
            }
          })
          // console.log("ROUTE ID IS: " + routeID)
        }}
        style={styles.touchable}
      >
        <View style={styles.route}>
          <Text style={styles.routetitle}>{title}</Text>
          <View style={styles.routebutton}>
            <Ionicons name={'arrow-forward-circle-outline'} size={40} />
          </View>
        </View>
      </TouchableHighlight>
    </View>
  )
};

function WayfindingPage(): JSX.Element {

  // array of data, 1 object per route displayed
  const DATA = [
    {
      id: '01-parking-clinic',
      title: 'Main Parking Lot to Pediatric Surgery Clinic',
    },
    {
      id: '02-parking-labs',
      title: 'Main Parking Lot to Lab',
    },
    {
      id: '03-hosp-enter-radiology',
      title: 'Hospital Entrance to Radiology',
    },
    {
      id: '04-hosp-enter-OR-checkin',
      title: 'Hospital Entrance to OR Check In',
    }
  ];

  return (
    <SafeAreaView style={styles.background}>
      <Text style={styles.headerText}> Wayfinding </Text>
      <FlatList
        data={DATA}
        renderItem={({ item }) =>
          <Route title={item.title} routeID={item.id} />
        }
        keyExtractor={item => item.id}
        style={styles.list}
      />
    </SafeAreaView>
  );
}

const styles = StyleSheet.create({
  background: {
    backgroundColor: "#003a5d",
  },
  list: {
    height: '100%'
  },
  headerText: {
    paddingTop: 30,
    paddingBottom: 20,
    textAlign: 'center',
    color: 'white',
    fontWeight: 'bold',
    fontSize: 36,
  },
  touchable: {
    backgroundColor: '#00b2e3',
    marginVertical: 5,
    marginHorizontal: 12,
    borderRadius: 10
  },
  route: {
    backgroundColor: '#00b2e3',
    paddingLeft: 20,
    flexDirection: 'row',
    flex: 2,
    justifyContent: 'space-between',
    alignContent: 'center',
    borderRadius: 10
  },
  routetitle: {
    fontSize: 18,
    color: 'white',
    fontWeight: 'bold',
    alignSelf: 'center',
    flex: 1,
    flexWrap: 'wrap'
  },
  routebutton: {
    alignSelf: 'flex-end',
    padding: 20
  }
});

export default WayfindingPage;