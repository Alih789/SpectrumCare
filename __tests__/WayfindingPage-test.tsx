/** @jest-environment jsdom */

import 'react-native';
import React from 'react';
import WayfindingPage from '../pages/WayfindingPage';
import renderer from 'react-test-renderer';
import { render, screen, fireEvent } from '@testing-library/react';

jest.mock('@react-navigation/native', () => ({
  useNavigation: jest.fn(),
}));

describe("WayfindingPage", () => {

  test("renders WayfindingPage", () => {
    const tree = renderer.create(
      <WayfindingPage />

    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});