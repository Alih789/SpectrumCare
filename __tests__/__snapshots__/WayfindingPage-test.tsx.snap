// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WayfindingPage renders WFCarousel 1`] = `
<RCTSafeAreaView
  style={
    {
      "backgroundColor": "#003a5d",
      "flex": 1,
    }
  }
>
  <Text
    style={
      {
        "color": "white",
        "fontFamily": "Figtree-SemiBold",
        "fontSize": 36,
        "paddingBottom": 10,
        "paddingTop": 20,
        "textAlign": "center",
      }
    }
  >
     Wayfinding 
  </Text>
  <RCTScrollView
    data={
      [
        {
          "id": "01-parking-clinic",
          "title": "Main Parking Lot to Pediatric Surgery Clinic",
        },
        {
          "id": "02-parking-labs",
          "title": "Main Parking Lot to Lab",
        },
        {
          "id": "03-hosp-enter-radiology",
          "title": "Hospital Entrance to Radiology",
        },
        {
          "id": "04-hosp-enter-OR-checkin",
          "title": "Hospital Entrance to OR Check In",
        },
      ]
    }
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={[]}
    style={
      {
        "height": "100%",
      }
    }
    viewabilityConfigCallbackPairs={[]}
  >
    <View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View>
          <View
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#00b2e3",
                "borderRadius": 10,
                "marginHorizontal": 12,
                "marginVertical": 5,
              }
            }
          >
            <View
              style={
                {
                  "alignContent": "center",
                  "backgroundColor": "#003a5d",
                  "borderBottomColor": "white",
                  "borderBottomWidth": 1,
                  "borderTopColor": "white",
                  "borderTopWidth": 1,
                  "flex": 2,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "paddingBottom": 10,
                  "paddingLeft": 20,
                  "paddingTop": 10,
                }
              }
            >
              <Text
                style={
                  {
                    "alignSelf": "center",
                    "color": "white",
                    "flex": 1,
                    "flexWrap": "wrap",
                    "fontFamily": "Figtree-Medium",
                    "fontSize": 22,
                  }
                }
              >
                Main Parking Lot to Pediatric Surgery Clinic
              </Text>
              <View
                style={
                  {
                    "alignSelf": "flex-end",
                    "padding": 10,
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  selectable={false}
                  style={
                    [
                      {
                        "color": "white",
                        "fontSize": 40,
                      },
                      undefined,
                      {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View>
          <View
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#00b2e3",
                "borderRadius": 10,
                "marginHorizontal": 12,
                "marginVertical": 5,
              }
            }
          >
            <View
              style={
                {
                  "alignContent": "center",
                  "backgroundColor": "#003a5d",
                  "borderBottomColor": "white",
                  "borderBottomWidth": 1,
                  "borderTopColor": "white",
                  "borderTopWidth": 1,
                  "flex": 2,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "paddingBottom": 10,
                  "paddingLeft": 20,
                  "paddingTop": 10,
                }
              }
            >
              <Text
                style={
                  {
                    "alignSelf": "center",
                    "color": "white",
                    "flex": 1,
                    "flexWrap": "wrap",
                    "fontFamily": "Figtree-Medium",
                    "fontSize": 22,
                  }
                }
              >
                Main Parking Lot to Lab
              </Text>
              <View
                style={
                  {
                    "alignSelf": "flex-end",
                    "padding": 10,
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  selectable={false}
                  style={
                    [
                      {
                        "color": "white",
                        "fontSize": 40,
                      },
                      undefined,
                      {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View>
          <View
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#00b2e3",
                "borderRadius": 10,
                "marginHorizontal": 12,
                "marginVertical": 5,
              }
            }
          >
            <View
              style={
                {
                  "alignContent": "center",
                  "backgroundColor": "#003a5d",
                  "borderBottomColor": "white",
                  "borderBottomWidth": 1,
                  "borderTopColor": "white",
                  "borderTopWidth": 1,
                  "flex": 2,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "paddingBottom": 10,
                  "paddingLeft": 20,
                  "paddingTop": 10,
                }
              }
            >
              <Text
                style={
                  {
                    "alignSelf": "center",
                    "color": "white",
                    "flex": 1,
                    "flexWrap": "wrap",
                    "fontFamily": "Figtree-Medium",
                    "fontSize": 22,
                  }
                }
              >
                Hospital Entrance to Radiology
              </Text>
              <View
                style={
                  {
                    "alignSelf": "flex-end",
                    "padding": 10,
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  selectable={false}
                  style={
                    [
                      {
                        "color": "white",
                        "fontSize": 40,
                      },
                      undefined,
                      {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View>
          <View
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#00b2e3",
                "borderRadius": 10,
                "marginHorizontal": 12,
                "marginVertical": 5,
              }
            }
          >
            <View
              style={
                {
                  "alignContent": "center",
                  "backgroundColor": "#003a5d",
                  "borderBottomColor": "white",
                  "borderBottomWidth": 1,
                  "borderTopColor": "white",
                  "borderTopWidth": 1,
                  "flex": 2,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "paddingBottom": 10,
                  "paddingLeft": 20,
                  "paddingTop": 10,
                }
              }
            >
              <Text
                style={
                  {
                    "alignSelf": "center",
                    "color": "white",
                    "flex": 1,
                    "flexWrap": "wrap",
                    "fontFamily": "Figtree-Medium",
                    "fontSize": 22,
                  }
                }
              >
                Hospital Entrance to OR Check In
              </Text>
              <View
                style={
                  {
                    "alignSelf": "flex-end",
                    "padding": 10,
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  selectable={false}
                  style={
                    [
                      {
                        "color": "white",
                        "fontSize": 40,
                      },
                      undefined,
                      {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
  <RCTSafeAreaView
    style={
      {
        "bottom": 10,
        "position": "absolute",
        "right": 10,
      }
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "height": 50,
          "opacity": 1,
          "width": 50,
        }
      }
      testID="expand-button"
    >
      <Image
        source={
          {
            "testUri": "../../../assets/images/extraIcons/notesIconPhotoshoped.png",
          }
        }
        style={
          {
            "height": 50,
            "resizeMode": "contain",
            "width": 50,
          }
        }
      />
    </View>
  </RCTSafeAreaView>
</RCTSafeAreaView>
`;

exports[`WayfindingPage renders WayfindingPage 1`] = `
<RCTSafeAreaView
  style={
    {
      "backgroundColor": "#003a5d",
      "flex": 1,
    }
  }
>
  <Text
    style={
      {
        "color": "white",
        "fontFamily": "Figtree-SemiBold",
        "fontSize": 36,
        "paddingBottom": 10,
        "paddingTop": 20,
        "textAlign": "center",
      }
    }
  >
     Wayfinding 
  </Text>
  <RCTScrollView
    data={
      [
        {
          "id": "01-parking-clinic",
          "title": "Main Parking Lot to Pediatric Surgery Clinic",
        },
        {
          "id": "02-parking-labs",
          "title": "Main Parking Lot to Lab",
        },
        {
          "id": "03-hosp-enter-radiology",
          "title": "Hospital Entrance to Radiology",
        },
        {
          "id": "04-hosp-enter-OR-checkin",
          "title": "Hospital Entrance to OR Check In",
        },
      ]
    }
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={[]}
    style={
      {
        "height": "100%",
      }
    }
    viewabilityConfigCallbackPairs={[]}
  >
    <View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View>
          <View
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#00b2e3",
                "borderRadius": 10,
                "marginHorizontal": 12,
                "marginVertical": 5,
              }
            }
          >
            <View
              style={
                {
                  "alignContent": "center",
                  "backgroundColor": "#003a5d",
                  "borderBottomColor": "white",
                  "borderBottomWidth": 1,
                  "borderTopColor": "white",
                  "borderTopWidth": 1,
                  "flex": 2,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "paddingBottom": 10,
                  "paddingLeft": 20,
                  "paddingTop": 10,
                }
              }
            >
              <Text
                style={
                  {
                    "alignSelf": "center",
                    "color": "white",
                    "flex": 1,
                    "flexWrap": "wrap",
                    "fontFamily": "Figtree-Medium",
                    "fontSize": 22,
                  }
                }
              >
                Main Parking Lot to Pediatric Surgery Clinic
              </Text>
              <View
                style={
                  {
                    "alignSelf": "flex-end",
                    "padding": 10,
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  selectable={false}
                  style={
                    [
                      {
                        "color": "white",
                        "fontSize": 40,
                      },
                      undefined,
                      {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View>
          <View
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#00b2e3",
                "borderRadius": 10,
                "marginHorizontal": 12,
                "marginVertical": 5,
              }
            }
          >
            <View
              style={
                {
                  "alignContent": "center",
                  "backgroundColor": "#003a5d",
                  "borderBottomColor": "white",
                  "borderBottomWidth": 1,
                  "borderTopColor": "white",
                  "borderTopWidth": 1,
                  "flex": 2,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "paddingBottom": 10,
                  "paddingLeft": 20,
                  "paddingTop": 10,
                }
              }
            >
              <Text
                style={
                  {
                    "alignSelf": "center",
                    "color": "white",
                    "flex": 1,
                    "flexWrap": "wrap",
                    "fontFamily": "Figtree-Medium",
                    "fontSize": 22,
                  }
                }
              >
                Main Parking Lot to Lab
              </Text>
              <View
                style={
                  {
                    "alignSelf": "flex-end",
                    "padding": 10,
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  selectable={false}
                  style={
                    [
                      {
                        "color": "white",
                        "fontSize": 40,
                      },
                      undefined,
                      {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View>
          <View
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#00b2e3",
                "borderRadius": 10,
                "marginHorizontal": 12,
                "marginVertical": 5,
              }
            }
          >
            <View
              style={
                {
                  "alignContent": "center",
                  "backgroundColor": "#003a5d",
                  "borderBottomColor": "white",
                  "borderBottomWidth": 1,
                  "borderTopColor": "white",
                  "borderTopWidth": 1,
                  "flex": 2,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "paddingBottom": 10,
                  "paddingLeft": 20,
                  "paddingTop": 10,
                }
              }
            >
              <Text
                style={
                  {
                    "alignSelf": "center",
                    "color": "white",
                    "flex": 1,
                    "flexWrap": "wrap",
                    "fontFamily": "Figtree-Medium",
                    "fontSize": 22,
                  }
                }
              >
                Hospital Entrance to Radiology
              </Text>
              <View
                style={
                  {
                    "alignSelf": "flex-end",
                    "padding": 10,
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  selectable={false}
                  style={
                    [
                      {
                        "color": "white",
                        "fontSize": 40,
                      },
                      undefined,
                      {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View>
          <View
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "backgroundColor": "#00b2e3",
                "borderRadius": 10,
                "marginHorizontal": 12,
                "marginVertical": 5,
              }
            }
          >
            <View
              style={
                {
                  "alignContent": "center",
                  "backgroundColor": "#003a5d",
                  "borderBottomColor": "white",
                  "borderBottomWidth": 1,
                  "borderTopColor": "white",
                  "borderTopWidth": 1,
                  "flex": 2,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "paddingBottom": 10,
                  "paddingLeft": 20,
                  "paddingTop": 10,
                }
              }
            >
              <Text
                style={
                  {
                    "alignSelf": "center",
                    "color": "white",
                    "flex": 1,
                    "flexWrap": "wrap",
                    "fontFamily": "Figtree-Medium",
                    "fontSize": 22,
                  }
                }
              >
                Hospital Entrance to OR Check In
              </Text>
              <View
                style={
                  {
                    "alignSelf": "flex-end",
                    "padding": 10,
                  }
                }
              >
                <Text
                  allowFontScaling={false}
                  selectable={false}
                  style={
                    [
                      {
                        "color": "white",
                        "fontSize": 40,
                      },
                      undefined,
                      {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
  <RCTSafeAreaView
    style={
      {
        "bottom": 10,
        "position": "absolute",
        "right": 10,
      }
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "height": 50,
          "opacity": 1,
          "width": 50,
        }
      }
      testID="expand-button"
    >
      <Image
        source={
          {
            "testUri": "../../../assets/images/extraIcons/notesIconPhotoshoped.png",
          }
        }
        style={
          {
            "height": 50,
            "resizeMode": "contain",
            "width": 50,
          }
        }
      />
    </View>
  </RCTSafeAreaView>
</RCTSafeAreaView>
`;
